Search/OptimizeableInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of the MassiveSearchBundle
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Massive\Bundle\SearchBundle\Search;
+
+interface OptimizeableInterface
```

Comment:
think i would call this `OptimizeableAdapterInterface`

Reply:
is okay for me did rename it.